examples/cactus-check-connection-ethereum-validator/check-ethereum-validator.ts

```diff
@@ -1,13 +1,10 @@
 import { Router, NextFunction, Request, Response } from "express";
 import { getLogger } from "log4js";
 import { TransactionManagement } from "../../packages/cactus-cmd-socketio-server/src/main/typescript/routing-interface/TransactionManagement";
-import {
-  RIFError,
-  BadRequestError,
-} from "../../packages/cactus-cmd-socketio-server/src/main/typescript/routing-interface/RIFError";
+import { RIFError } from "../../packages/cactus-cmd-socketio-server/src/main/typescript/routing-interface/RIFError";
```

Comment:
@outSH Could this be replaced with the official package name instead? The way you are importing through the direct file system path will not work when the package is installed through npm instead via the local development symlink setup.

Reply:
@petermetz I'd rather not change it in this PR, entire package use path imports and I'd have to change them all. I've just automatically removed unused import when I saw it, didn't add anything new from cactus-cmd-socketio-server :)